nautobot/dcim/views.py

```diff
@@ -247,8 +247,8 @@ def get_extra_context(self, request, instance):
             .select_related("parent", "location_type")
         )
 
-        children_table = tables.LocationTypeTable(children)
-        locations_table = tables.LocationTable(locations)
+        children_table = (tables.LocationTypeTable, children)
+        locations_table = (tables.LocationTable, locations)
```

Comment:
I was hoping we could take this a step further and have a pattern similar what you've got with StatsPanel now, something like:

```
object_detail_content = ...
   ObjectsTablePanel(table_class=tables.LocationTypeTable, filter_field="parent")
   ObjectsTablePanel(table_class=tables.LocationTable, filter_field="location_type"),
```